.github/workflows/complexity-check.yml

```
@@ -25,19 +25,21 @@ jobs:
-          TOTAL_TEST_LINES=$(find tests -name "*.js" -exec wc -l {} + 2>/dev/null | tail -1 | awk '{print $1}' || echo 0)
+          # Count total lines in streamlined test framework (excluding E2E tests)
+          TOTAL_TEST_LINES=$(find tests -name "*.js" -not -path "tests/e2e/*" -exec wc -l {} + 2>/dev/null | tail -1 | awk '{print $1}' || echo 0)
           
-          # Count test files
-          TEST_FILE_COUNT=$(find tests -name "*.js" | wc -l || echo 0)
+          # Count test files (excluding E2E tests)
+          TEST_FILE_COUNT=$(find tests -name "*.js" -not -path "tests/e2e/*" | wc -l || echo 0)
           
           # Count utility files (should be minimal)
           UTIL_FILE_COUNT=$(find tests \( -name "helpers.js" -o -name "setup.js" \) -type f 2>/dev/null | wc -l || echo 0)
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)